app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,94 @@
+class FeedbacksController < ApplicationController
+  layout "feedback_layout"
+
+  def new
+    path = validate_feedback(params)
+    unless path.nil?
+      return redirect_to path
+    end
+
+    # @order = Order.find(params[:order_id])
```

Comment:
commented code ?

Reply:
This is pending feature basically. It was added to check whether a customer has visited a link for review or not. I am removing it btw for now.